src/AppShed/Extensions/SpreadsheetMapsBundle/Controller/MapsController.php

```diff
@@ -235,12 +226,12 @@ private function getFilterString($filter, Request $request)
         return $str;
     }
 
-    private function getAroundMeQuery($distance, Request $request)
+    private function getAroundMeQuery($distance)
     {
-        $center = [
-            'lat' => $request->query->get('userlat', 0),
-            'lng' => $request->query->get('userlng', 0)
-        ];
+        $center = array(
+            'lat' => isset($_GET['userlat']) ? $_GET['userlat'] : 0,
```

Comment:
There are some changes here that we did in the new one, that you seem to have change back.